pkg/virt-handler/node-labeller/node_labeller.go

```diff
@@ -249,11 +249,11 @@ func (n *NodeLabeller) prepareLabels(node *v1.Node, cpuModels []string, cpuFeatu
 		newLabels[kubevirtv1.HypervLabel+key] = "true"
 	}
 
-	if c, err := n.capabilities.GetTSCCounter(); err == nil && c != nil {
+	if c := n.GetTSCCounter(); c != nil {
 		newLabels[kubevirtv1.CPUTimerLabel+"tsc-frequency"] = fmt.Sprintf("%d", c.Frequency)
 		newLabels[kubevirtv1.CPUTimerLabel+"tsc-scalable"] = fmt.Sprintf("%v", c.Scaling)
-	} else if err != nil {
-		n.logger.Reason(err).Error("failed to get tsc cpu frequency, will continue without the tsc frequency label")
+	} else {
+		n.logger.Error("failed to get tsc cpu frequency, will continue without the tsc frequency label")
```

Comment:
I think it might be worth to remove the log entry in its entirety, it kind of is a precedence (save for `isNodeRealtimeCapable` which should be taken care of as part of https://github.com/kubevirt/kubevirt/pull/12522.

Reply:
>  (save for isNodeRealtimeCapable which should be taken care of as part of #12522).

I don't follow this part, could you please elaborate?